stock_available/__openerp__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Stock available to promise',
-    'version': '9.0.1.0.0',
+    'version': '9.0.3.0.0',
```

Comment:
I'm talking about the normal version increment numbering: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers. As the version numbering has been rest in 9.0, you cannot jump from 9.0.1.0.0 to 9.0.3.0.0 in one PR. This happens because the cherry-pick.